doc/lsst.meas.algorithms/index.rst

```diff
@@ -8,6 +8,16 @@ lsst.meas.algorithms
 
 The ``lsst.meas.algorithms`` module provides a variety of astronomical measurement algorithms.
 
+.. _lsst.meas.algorithms-refcats:
+
+LSST Reference Catalogs
+=======================
+
+.. toctree::
+   :maxdepth: 1
+
+   creating-a-reference-catalog
+
```

Comment:
Add a contents listing for the tasks:

```rst
.. _lsst.meas.algorithms-taskref:

Task reference
==============

.. _lsst.meas.algorithms-tasks:

Tasks
-----

.. lsst-tasks::
   :root: lsst.meas.algorithms
   :toctree: tasks
```

Reply:
Should I see the task appear on this page once I've build the docs? Right now I'm getting "No topic".

Reply:
Uh oh. I'm a bit stumped on that one. If you want you can merge this and create a follow-on ticket for me to fix that.